assets/scss/learndash-styling.scss

```diff
@@ -366,4 +366,26 @@ nav.learndash-pager {
 	.featured-top {
 		display: none;
 	}
-}
\ No newline at end of file
+}
+
+.ld-course-list-items .ld_course_grid .thumbnail.course .ld_course_grid_price {
+	background: $color_nhsuk-green  !important;
+	color: $color_nhsuk-white;
+	position: absolute;
+	font-size: 14px;
+	left: -8px;
+	top: 10px;
+	padding: 3px 10px;
+	z-index: 2;
+	font-weight: bold;
+}
+
+.ld-course-list-items .ld_course_grid .thumbnail.course .ld_course_grid_price:before {
```

Comment:
in SASS file any classes with specificity traversing should be written vertically not horizontally eg: 
```
.ld-course-list-items { 
   .ld_course_grid {
      .th umbnail.course {
        .ld_course_grid_price:before {
        }
      }
   }
}
```

Reply:
Also we use list-items class already in SASS file line 260. Following SASS best practices will let you to avoid code redundancy as well as next time you will extend the existing structure not paste it